enterprise/backend/test/metabase_enterprise/models/entity_id_test.clj

```diff
@@ -60,6 +60,7 @@
     :model/PermissionsGroupMembership
     :model/PermissionsRevision
     :model/PersistedInfo
+    :model/Pulse
```

Comment:
This was tragic. It feels like a test should have caught this, perhaps something like testing that this is the union of `v2.models/excluded-models` and `inlined-models` (and that those are disjoint)?

Reply:
uhhh this is hard and a bit disjointed :\